### ISSUE TITLE
Cut 1.20.3 release

### ISSUE THREAD

**Issue by cpanato** [kubernetes/sig-release#1467]
## Scheduled to happen: Wed, 2021-02-17

- [x] Screenshot unhealthy release branch testgrid boards and add them as a comment
- [ ] Stage & release
  - [x] Notify [#release-management](https://kubernetes.slack.com/messages/CJH2GBF7Y): [see notification](https://kubernetes.slack.com/archives/CJH2GBF7Y/p1613553511289200)
  - [x] Collect metrics and add them to the `Release steps` table below
  - [ ] Build & publish packages (debs & rpms)
- [ ] Send notification [email](https://link-to-email-on-kubernetes-announce-google-group)

## Release steps

[krel](https://github.com/kubernetes/release/tree/master/docs/krel) version:
```
GitVersion:    v0.7.0
GitCommit:     eb6492dcdb683c53e6885ad5aecdc0d055544788
GitTreeState:  clean
BuildDate:     2021-02-17T09:30:49Z
GoVersion:     go1.15.8
Compiler:      gc
Platform:      linux/amd64
```

| Step | Command | Link | Start | Duration | Succeeded? |
| --- | --- | --- | --- | --- | --- |
| `mock stage`   | `krel stage --type official --branch release-1.20 --build-version v1.20.3-rc.0.77+e4f2398716b35f`            | https://console.cloud.google.com/cloud-build/builds/a9517307-1623-4006-a805-9505dceaa264?project=648026197307 | 2021-02-17T09:44:24.538234790Z | 00:01:23 | No         |
| `mock stage`   | `krel stage --type official --branch release-1.20 --build-version v1.20.3-rc.0.77+e4f2398716b35f`            | https://console.cloud.google.com/cloud-build/builds/c8144aca-7543-4afa-9e8c-c1910e02c782?project=648026197307 | 2021-02-17T10:39:17.511267249Z | 01:17:49 | Yes        |
| `mock release` | `krel release --type official --branch release-1.20 --build-version v1.20.3-rc.0.77+e4f2398716b35f`          | https://console.cloud.google.com/cloud-build/builds/53520481-5bbd-49bf-b9f9-2a24b4495e84?project=648026197307 | 2021-02-17T12:20:52.240586669Z | 00:06:31 | Yes        |
| `stage`        | `krel stage --type official --branch release-1.20 --build-version v1.20.3-rc.0.77+e4f2398716b35f --nomock`   | https://console.cloud.google.com/cloud-build/builds/7b7ca999-731c-46d2-b6b1-48053a9d714f?project=648026197307 | 2021-02-17T12:30:13.520951825Z | 01:17:43 | Yes        |
| Image Promotion | ` krel promote-images -i --fork=cpanato --tag=v1.18.16,v1.18.17-rc.0,v1.19.8,v1.19.9-rc.0,v1.20.3,v1.20.4-rc.0` | https://github.com/kubernetes/k8s.io/pull/1670 |  |  |  |
| `release`      | `krel release --type official --branch release-1.20 --build-version v1.20.3-rc.0.77+e4f2398716b35f --nomock` | https://console.cloud.google.com/cloud-build/builds/ddc2918a-1677-4acc-9e7f-7583d709f797?project=648026197307 | 2021-02-17T20:30:04.043334220Z | 00:06:48 | Yes        |
| Cut packages  | -- | -- |  |  |  |
| Notify | -- |  |  | -- |  |

## Open questions

/milestone v1.20
/cc @kubernetes/release-managers
/priority important-soon
/kind documentation

**Comment by cpanato** [comment]
### Testgrid dashboards for 1.20

Boards checked for FAILING:
- [sig-release-1.20-blocking](https://testgrid.k8s.io/sig-release-1.20-blocking)
- [sig-release-1.20-informing](https://testgrid.k8s.io/sig-release-1.20-informing)


<details><summary><tt>2021-02-17 09:38:42.838860841 +0000 UTC</tt> FAILING <a href="https://testgrid.k8s.io/sig-release-1.20-informing#gce-windows-1909-1.20&width=30">sig-release-1.20-informing#gce-windows-1909-1.20 - TestGrid</a></summary><p>

![sig-release-1.20-informing#gce-windows-1909-1.20](https://storage.googleapis.com/k8s-staging-releng/testgridshot/1.20/2021-02-17-3bc2e39b-d162-400c-a22f-8f734eafc324/sig-release-1.20-informing-gce-windows-1909-1.20-FAILING.jpg)

</p></details>
<details><summary><tt>2021-02-17 09:38:42.838860841 +0000 UTC</tt> FAILING <a href="https://testgrid.k8s.io/sig-release-1.20-informing#gce-windows-2019-1.20&width=30">sig-release-1.20-informing#gce-windows-2019-1.20 - TestGrid</a></summary><p>

![sig-release-1.20-informing#gce-windows-2019-1.20](https://storage.googleapis.com/k8s-staging-releng/testgridshot/1.20/2021-02-17-3bc2e39b-d162-400c-a22f-8f734eafc324/sig-release-1.20-informing-gce-windows-2019-1.20-FAILING.jpg)

</p></details>




**comment generated by [krel](https://github.com/kubernetes/release/tree/master/docs/krel)**

**Comment by cpanato** [comment]
> A post-release change to the v1.20 conformance metadata was discovered, which has now been fixed.
Details here: https://github.com/kubernetes/kubernetes/pull/99081
> Unfortunately, the revert happened after we started the release process today, which means we need to cut a new release to fix conformance tests for consumers.

Stephen's email: https://console.cloud.google.com/cloud-build/builds/f5e98f4b-7c2c-49f8-91bf-87c206346586?project=kubernetes-release-test

Slack 🧵  https://kubernetes.slack.com/archives/C78F00H99/p1613501041053200

**Comment by cpanato** [comment]
This will be close and a new issue to cut a new patch release will be created

/close

**Comment by oldthreefeng** [comment]
Is this version available for production  use ? or use 1.20.4 instead .  @cpanato

**Comment by cpanato** [comment]
@oldthreefeng  use 1.20.4, please see: https://groups.google.com/g/kubernetes-dev/c/oUpY9vWgzJo
